pkg/daemon/update.go

```diff
@@ -792,6 +807,94 @@ func (dn *Daemon) switchKernel(oldConfig, newConfig *mcfgv1.MachineConfig) error
 	return nil
 }
 
+func addExtRepo(mnt string) error {
+	if err := os.MkdirAll(filepath.Dir(extensionsRepo), 0755); err != nil {
+		return fmt.Errorf("Error creating dir %s: %v", filepath.Dir(extensionsRepo), err)
```

Comment:
maybe "Error created extensions repo:.."

Reply:
Fixed with "error creating yum repo directory" as we are creating standard `/etc/yum.repos.d` where rpm-ostree looks for repos.